frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java

```diff
@@ -111,6 +111,10 @@ private void initModelStore() {
         }
 
         ModelManager modelManager = ModelManager.getInstance();
+        int workers = configManager.getNumberOfGpu();
```

Comment:
Why are workers tied to number of GPUs? They should be independent.

Reply:
What should be default number of worker when server startup?
Isn't this the right behavior?

Reply:
We should let MMS users configure this. If they want 1 worker we create only 1 worker.  Falling back to num-gpu's should be used as only if customers want us to fallback (customers should be given an option to choose this as well). The intuition for this is, num-gpu number of workers isn't the most efficient number of workers for most cases. If anything, this should be num-vcpus number of workers, but that might fail if we try to load all these workers models onto GPUs. Example of this could be if a host has num-vcpus is 64 and num_gpus is 1 with 8GB and each model worker requires 1GB of memory. Since there is no magic bullet solution yet to pick the right number of workers, we should let customers decide with a fallback option to num_gpus or num_cpus.